solidity-frontend/src/test/java/org/sonarsource/solidity/frontend/UtilsTest.java

```diff
@@ -0,0 +1,5 @@
+package org.sonarsource.solidity.frontend;
+
+public class UtilsTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=23&issues=AWQ2bn0H_bEzQ-kUlT1W&open=AWQ2bn0H_bEzQ-kUlT1W)

[ISSUE_KEY]: AWQ2bn0H_bEzQ-kUlT1W